rasterio/gdal.pxi

```diff
@@ -166,6 +166,7 @@ cdef extern from "ogr_srs_api.h" nogil:
                      double *y, double *z)
     void OSRCleanup()
     OGRSpatialReferenceH OSRClone(OGRSpatialReferenceH srs)
+    OGRSpatialReferenceH OSRCloneGeogCS(OGRSpatialReferenceH srs)
```

Comment:
Just checking that this goes back to GDAL 3.5.0 or earlier?

Reply:
looks like it is in 3.5: 

https://github.com/OSGeo/gdal/blob/62565af02dd939ad5d05d95b630b6dd87c012010/ogr/ogr_srs_api.h#L476,

maybe dates back to 1.6: https://github.com/OSGeo/gdal/commit/5dedc15a17b9850ca50aad88c50821eb519c0a0d